modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -174,6 +180,23 @@
 				[b]Note:[/b] If [member ProjectSettings.editor/export/convert_text_resources_to_binary] is [code]true[/code], [method @GDScript.load] will not be able to read converted files in an exported project. If you rely on run-time loading of files present within the PCK, set [member ProjectSettings.editor/export/convert_text_resources_to_binary] to [code]false[/code].
 			</description>
 		</method>
+		<method name="ord">
+			<return type="int" />
+			<param index="0" name="char" type="String" />
+			<description>
+				Returns an integer representing the Unicode code point of the given character [param char] (as a [String] of length 1).
+				[codeblock]
+				a = ord("A") # a is 65
+				a = ord("€") # a is 8364
+
+				# 0x20AC == 8364
+				a = ord("\u20AC") # a is 8364
+				a = 0x20AC        # a is 8364
+				[/codeblock]
+				This is the inverse of [method char]. See also [method String.chr] and [method String.unicode_at].
+				[b]Note:[/b] Unicode is compatible with ASCII for the first 128 characters.
```

Comment:
This note may seem good at first, but for most users reading this it doesn't add much of substance, for what it may actually be used for in practice. At least, that's how I feel. Long story short, for this note to be useful users have got to manually look up what the first 128 Unicode characters are, which will lead them to discover what ASCII is in the first place.